vignettes/estimate_ascertainment.Rmd

```diff
@@ -226,8 +226,11 @@ We adopt data science tools to iteratively apply the `estimate_ascertainment()`
 We refer the user to the book [R for Data Science](https://r4ds.had.co.nz/) for a better explanation of some of the code used here, including from the packages in [the Tidyverse](https://www.tidyverse.org/).
 
 ```{r}
+# subset for early covid outbreaks
+covid_data_early <- filter(covid_data, date < "2020-06-01")
+
 # nest the data
-df_reporting <- nest(covid_data, .by = country)
+df_reporting <- nest(covid_data_early, .by = country)
```

Comment:
This looks more plausible for most countries, but looking at the remaining 100% estimates (i.e. Spain, Germany, France, Ukraine), it's happening because the data for those four countries are only weekly, but the underascertainment algorithm (and underlying CFR calculation) is designed for daily data, which is why it's returning an erroneous result. I'd therefore suggest we omit these countries with a brief note to user about why?

Reply:
Oh, so that's what's going on - I thought it might be that they actually have really good reporting. I'll exclude these countries from the vignettes, but are they perhaps worth keeping in the dataset? Is there a way to modify the current methods to deal with low-frequency reporting?

Reply:
Converted to issue - happy to discuss further!